src/lib/utils/number.ts

```diff
@@ -22,3 +26,15 @@ export const formatByDecimalPlace = (
 	const number = Number(value);
 	return Number.isNaN(number) ? '' : number.toLocaleString(localeCode, config);
 };
+
+// Formatted highest bid in WILD
+export const getFormattedHighestBidAmount = (highestBid?: string) =>
+	highestBid
+		? ethers.utils.formatEther(highestBid).toString() + ` ${CURRENCY.WILD}`
+		: '';
+
+// Formatted bid amount in WILD
+export const getFormattedBidAmount = (bidAmount?: string) =>
+	bidAmount
+		? ethers.utils.formatEther(bidAmount).toString() + ` ${CURRENCY.WILD}`
+		: '';
```

Comment:
nit: looks like these two are the same function - could just change it to `formatBidAmount`